PeerCo/small_scale_self_consumed_solar.md

```diff
@@ -0,0 +1,289 @@
+1. the maximum PV generation is in line with the PV asset's nominal rating,
+2. self-consumption is always lower than the total consumption,
+3. no significant PV generation is happening when there is no solar irradiance (e.g. at night).
+
+Where data is missing, filling of the gaps sall be allowed under the following conditions:
+1. data is not missing more than 5% of the time within a calendar month,
+2. gaps are filled with data whose daily profile and magnitude is consistent with the rest of the asset generation,
+3. if accumulated values are available for the relevant energy measurements, the sum of the generation over the missing data periods shall equate the generation indicated in the accumulated measurements.
+
+When carrying out this asset data verification, a tolerance should be given for numerical purposes (e.g. to account for rounding of measurements by digital systems). This tolerance shall however not be of a magnitude that significantly impacts the volume of the certificates generated.
+
+### Emission intensity data
+
+The emission intensity data used in calculations shall meet the following requirements:
+
+1. the time resolution of the data is hourly or higher,
+2. the geographical resolution of the data is country-level or higher,
+3. the signal corresponds to an average emission intensity of *consumed* electricity,
+4. the source of the carbon intensity data shall be recorded as part of the digital asset generated.
+
+Data may originate from one of the sources listed in *Appendix 2*.
+
+## Verification and additionality
+
+This section provides the key aspects of the verification methodology, based upon the data of the previous section.
+
+### Baseline scenario
+
+The baseline scenario considered is a scenario where the PV asset is not installed, therefore zero generation happens from this asset and electricity is imported from the grid instead.
+
+### Quantification of GHG emission reduction and/or clean energy produced
+
+This methodology supports the generation of EACs accounting for the energy consumed on site, which also account for the avoided carbon emissions (as an extra information embedded in the certificates).
+
+The clean energy generation accounted in this methodology shall be limited to the self-consumed part of the PV asset (i.e. exports are thereby excluded, as they may already be accounted for in another certificate scheme).
+
+Certificates shall be generated for specific time periods, where the length of these time periods shall not exceed one hour. In contexts where the electricity market's settlement period is less than an hour (e.g. 30 minutes for Great Britain), the certificates may be issued for durations matching this settlement period.
+
+For each period *i*, the certificate's energy content shall be:
+> EAC energy content[i] [kWh] = Self-consumed PV asset generation[i] [kWh]
+
```

Comment:
It seems to me that the offset should be based on the difference between the amount of grid electricity that would have been consumed in the counterfactual no-solar-panel world, and which is not consumed because of the installation.   

Is that number the same number as the total self-consumption?  I think it could be, but I could also imagine a customer self-consuming more electricity than they otherwise would have without panels.  You could even end up with a perverse incentive where people deliberately self-consume more and more electricity to fraudulently generate larger credits.  Have you thought about checking for this?

Reply:
We are assuming there are other incentives at play that make the maximising for self-consumption aligned with those other incentives. If they have a battery, they will also be trying to use that and discharge rather than using the grid. If they have an electric car, they may be trying to charge it. For SMEs the idea of having contextual data so that we can show their per unit energy and carbon intensity (and that it is going down) will also be possible.

Reply:
@paulteehan - the counterfactual in this methodology is indeed assumed to be grid import for the whole value for the PV production (i.e. we assume that no demand change happens following the installation of PV - neither in amount or time of demand).

Here, we need to remember that the credits are either used in reporting or trading. 
- When *reporting*, they wouldn't really profit from more demand, as that's a cost implying more import or less export (while no profit from EAC sale). 
- For *trading*, the same applies (bearing in mind that the EAC value is likely going to be significantly under the kWh value for the electricity itself - so that they would actually make a loss by gaming). On top of this, entities that need to report would therefore give away the 'greenness' of that demand (as the attribute is sold) which would then result in degrading their carbon reporting further and have even less incentive to game it.

Does that make sense to you?

Reply:
We discussed this in the open EAC alliance meeting today and the following points came through:

- It's not obvious if the assumption of no demand change following PV installation is valid or not. 
- Because these EACs will be used as carbon offsets, any optimistic over-counting of emissions reductions will result in double-emissions so it's important to get the numbers right
- The group recognizes that the burden of evaluating counterfactuals is significant, especially if you get into the full protocol of 12 months of baseline data, max bias and error thresholds, non-routine event adjustment, etc.   Some people think this analysis is necessary and some take a more pragmatic approach to getting the projects done.  
- Are there studies that exist which could validate the assumption?
- Perhaps this could be an issue for residential but not commercial customers
- Perhaps customers that make material changes e.g. EV installation following installation should be excluded

Reply:
Thanks for the summary of your subgroup's reflection yesterday. We definitely agree that counterfactual building is a difficult task where the method and outcome depend on the application case (eg. residential, industrial process, commercial,...). Without resorting to a full-scale literature review, the following demand changes can be named: demand flexibility to match local production (which in practice is a combination of PV installation and the introduction of a demand control scheme), further electrification on site (EV, heat pump, ... - where again a combination of actions is happening), increase in demand (either due to rebound effect on power cost, or production expansion for industry). While some of these are a combination of actions, where solar might have acted as a trigger, it isn't clear how and where to draw the line of the solar-specific change. 

Having said that, and witnessing the agreement on the complexity of the matter, there's a question whether we could specify counterfactual models precisely enough to be transparent while covering most cases. Another question arising then is whether this would reach a methodology size, consensus level and complexity that is compatible with our review process. On this level, we would expect that the answer is far from a 'yes' within a reasonable time horizon. 

This is why we propose to stand by the current counterfactual, maybe making it more explicit so that it doesn't appear to be a blind spot but rather a mindful choice. The specification of this counterfactual could be a more central/defining element of this methodology (eg by also adjusting the title to signal this), so that it'd be clear and transparent to users and EAC buyers. For applications where counterfactual building can be specified and agreed upon more easily, we should promote the creation of new versions as opportunities arise. 


From a tradeable instrument perspective (rather than carbon credit), it is also worth bearing in mind that the counterfactual might need to be thought of differently and more from a market based approach. Specifically here, what the EACs represent is the solar power that was used on a site (rather than importing) which is transacted with another party as a way to 'swap' their import of power with the solar production on the other site. Adopting this perspective does raise much fewer questions as to the validity of the current counterfactual. 

However, this is more of an EAC discussion than an M&V one.


Please let us know what your thoughts are on the above.

Reply:
Hi @Pierre-VF , I'm happy to give my personal perspective on this approach, with the caveat that I'm not sure what happens when I do - do we have to reach a kind of consensus in order to move the method forward, or is it more that you might take my comments under advisement?   I might make a critique in the interests of maximizing rigour, but I wouldn't necessarily want to be an obstacle, and I'm just a practitioner with a perspective which is as valid as anyone else's here.   Perhaps a question for someone from WattCarbon (@ssuffian ?)

My opinion is that if you have factors in your equation that are uncertain or unknown, such as rebound effects, additional electrification, etc, then you need to model these as accurately as you can, and issue a credit which is conservatively on the low end of your range to account for the unknowns.   (This also holds for life cycle impacts mentioned above.)   I don't think that measurement complexity or difficulty in obtaining good numbers are reasons to conclude that factors can be ignored completely or assumed to be zero.  Rather these are arguments that an accurate calculation cannot be made at all, and therefore we should not be issuing credits at all.    I do agree with the point about tradeable instruments; I would hold offsets or credits to a far higher methodological standard since they are associated with actual physical emissions and if the numbers are found to be overly optimistic that could undermine the integrity of the market and the goal of decarbonization itself.  

If this was my project, my next step would be to try to get hold of some pre/post data for a sample of customers and see if I could measure the change in demand and determine whether increased self-consumption perfectly matches decreased demand, and if it's off, by how much.  Or, perhaps find some reasonably proxies in the literature.  Then I would include that error in my equations so that I'm reporting a pessimistic number.